web_warning_on_save/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Raise warning when saving
+=========================
+
+This module was written to extend the functionality of saving a record in the web interface.
+/!\/!\/!\ In no way this module stops the save of the record. You must consider this as a warning displayed 
+to the user AFTER save completed. /!\/!\/!\
+If you don't want OpenERP to save the record, you should use constraints.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* write a method called 'check_warning_on_save' which will make some checks and return a string
+
+example :
```

Comment:
In my case, I believe I had some non-shown character which was causing all the trouble.
I solved it removing the "=" underline in offending title, with previous and following line-feed, and type it again.